apps/dashboard/src/pages/group.tsx

```diff
@@ -479,6 +508,28 @@ ${memberIds.join("\n")}
                             )}
                         </Box>
                     )}
+                    {_group.type === "off-chain" && isGroupAdmin && (
+                        <Box
+                            bgColor="balticSea.50"
+                            p="25px 30px 25px 30px"
+                            borderRadius="8px"
+                        >
+                            <Text fontSize="20px">Download group</Text>
+                            <Flex align="center" mt="2">
+                                <Text my="10px" fontWeight="400">
+                                    Get the group&apos; data JSON format.
```

Comment:
```suggestion
                                    Get the group&apos;s data JSON format.
```
The s is missing here.

Or what about just adding the following to make it easier:

```suggestion
                                    Get the group data in JSON format.
```